hypertrace-trace-enricher/hypertrace-trace-enricher/build.gradle.kts

```diff
@@ -46,10 +46,16 @@ dependencies {
     implementation("com.google.guava:guava:30.1-jre") {
       because("Information Disclosure [Medium Severity][https://snyk.io/vuln/SNYK-JAVA-COMGOOGLEGUAVA-1015415] in com.google.guava:guava@29.0-android")
     }
+    runtimeOnly("io.netty:netty-codec-http2:4.1.60.Final") {
+      because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-1083991")
+    }
+    runtimeOnly("io.netty:netty-handler-proxy:4.1.60.Final") {
+      because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-1083991")
+    }
   }
 
   // Required for the GRPC clients.
-  runtimeOnly("io.grpc:grpc-netty-shaded:1.35.0")
+  runtimeOnly("io.grpc:grpc-netty:1.35.0")
```

Comment:
also a bit of cleanup here - we don't shade netty in other deployments.

Reply:
totally speaking out of ignorance (maybe unrelated here). Is there any action that tells us how bigger/smaller the artifact or docker image got after a change in PR? cc @JBAhire

Reply:
I suspect there's one out there, yeah (and if not, certainly seems feasible to build a trivial one). Was curious and just did a quick compare on the ht-trace-enricher image and there's a small improvement here: 
![image](https://user-images.githubusercontent.com/45047841/110853514-93094500-8268-11eb-8b8b-948b70337ebc.png) (a89e306d4a74 is a build of current main, 4596e35d6c9c this branch)